src/transmuxer-worker.js

```diff
@@ -155,6 +155,18 @@ const wireTransmuxerEvents = function(self, transmuxer) {
     });
   });
 
+  transmuxer.on('debug', function({message}) {
+    self.postMessage({action: 'debug', message});
+  });
+
+  transmuxer.on('error', function({message}) {
+    self.postMessage({action: 'error', message});
+  });
+
+  transmuxer.on('warn', function({message}) {
+    self.postMessage({action: 'warn', message});
+  });
```

Comment:
I know the mux.js commit is already in, but I think we may be better off using `log` with a level and message. `error` usually is a special case separate from standard log messages, and it would also allow us to only have one function all the way through the chain.